deployment/environment_min.yml

```diff
@@ -20,8 +20,8 @@ dependencies:
   - deepdish
   - fenics
   - kwant
-  - mkl-service
-  - openblas # fenics needs this, but somehow fenics didn't include this as a dependency when conda forge moved to mkl
+  - libblas=*=*mkl
+  - openblas  # because of https://github.com/conda-forge/fenics-feedstock/issues/82
```

Comment:
We can remove this line once https://github.com/conda-forge/fenics-feedstock/pull/83 is merged.